Lib/test/test_capi/test_getargs.py

```diff
@@ -588,12 +591,12 @@ def test_tuple(self):
         self.assertEqual(ret, (1,2,3))
 
         # make sure invalid tuple arguments are handled correctly
```

Comment:
```suggestion
        # make sure invalid sequence arguments are handled correctly
```

Should the comment not also be changed?